packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -59,7 +59,7 @@ const { details: X } = assert;
 
 /**
  * @param {import('./vaultFactory.js').VaultFactoryZCF} zcf
- * @param {import('@agoric/governance/src/contractGovernance/typedParamManager').TypedParamManager<{Electorate: "invitation"}>} directorParamManager
+ * @param {import('@agoric/governance/src/contractGovernance/typedParamManager').TypedParamManager<{Electorate: "invitation", LiquidationInstall: "installation", LiquidationTerms: "unknown", MinInitialLoan: "amount"}>} directorParamManager
```

Comment:
This seems like it should also be `VaultDirectorParams` above.

Reply:
tried that.  I'll leave it to @turadg to fix.

Reply:
https://github.com/Agoric/agoric-sdk/pull/5243 has the fix. Feel free to click merge on that if you want it in this PR or merge this PR and I'll land the TS fix on master.